### DISCUSSION TITLE
Does this work on the latest LTSC windows 10 version?

### DISCUSSION THREAD

**Discussion by aeon3** [simeononsecurity/Windows-Optimize-Harden-Debloat#45] [Q&A] [answered]
Hey I just got this fresh LTSC installation and I wanted to start with a good privacy enhancing script. LTSC is not mentioned in the supported windows version, but I wanted to doublecheck.

[a comment by an automated account is not shown]

**Comment by simeononsecurity** [comment, marked as answer]
> Hey I just got this fresh LTSC installation and I wanted to start with a good privacy enhancing script. LTSC is not mentioned in the supported windows version, but I wanted to doublecheck.

LTSC branches practically are just Windows with the security patches but none of the feature upgrades from current date minus two years. Yes, it should work, however we don't test the LTSC branch. YMMV.
